mteb/tasks/Clustering/eng/BiorxivClusteringP2P.py

```diff
@@ -10,7 +10,7 @@
 
 class BiorxivClusteringP2PFast(AbsTaskClusteringFast):
     metadata = TaskMetadata(
-        name="BiorxivClusteringP2P.v2",
+        name="BiorxivClusteringP2P.v3-2",
```

Comment:
Yeah those are meant to be temporary and easy to tell apart since 4 options are presented here. We will keep v2 at the end of the discussion for sure.